### ISSUE TITLE
Vimeo Playback: Retrieve full stream

### ISSUE THREAD

**Issue by kila58** [webrecorder/browsertrix-crawler#632]
The crawler is working great, but struggles with Vimeo embeds that stream the video in chunks while playing. It will capture the first few seconds of a Vimeo embed as if it just pressed play, but the rest of the video fails to load as it hasn't buffered fully.

Example: [https://wewillteachyouleague.com/](url)

I know this is a pretty specific use case, so alternatively could I use behaviors to interact with the embed and load it fully? I've tried previously using a browser extension but I was struggling with puppeteer's support for that.

**Comment by ikreymer** [comment]
Yes, this is something we'd like to improve, and something that we'll likely have better support for in the future.

For now, if you know how long the video is, you can add a wait for the video to finish playing, eg: `--postLoadDelay 35`, I checked and that works for the page above for example, as its a ~35 second video.

This won't give you a single stream, but should include all of the chunks.
